readme.txt

```diff
@@ -29,6 +29,8 @@ Installing this plugin provides a new Intercom settings page, which allows you t
 
 If `current_user` is present, their email will be used as an identifier in the widget. Otherwise, it falls back to anonymous chat using [Acquire](https://www.intercom.io/live-chat).
 
+NB: This plugin injects a Javascript snippet on your website frontend containing dynamic user data. Some caching solutions will cache entire pages and should not be used with this plugin.
```

Comment:
Perhaps we should explicitly explain that it may show conversations to other users?